src/main/java/com/climatetree/places/model/PlaceInfo.java

```diff
@@ -167,6 +190,23 @@ public void setPointY(double pointY) {
 		this.pointY = pointY;
 	}
 
+
+	public PlaceDTO convertToPlaceDTO() {
```

Comment:
Can we move this to a mapper class ? Ideally models classes are best if they remain POJOs and so we could clearly exclude testing them since there's no business logic involded.

You could create a mapper class for all mapping functions for future reference and put it in a util package and then refactor your existing test for this to be testing that mapper class functionality.

Reply:
I agree with this structure. I'll make the changes and have you review again